lms/templates/main.html

```diff
@@ -85,30 +92,7 @@
 
   <%block name="headextra"/>
 
-<%
-  if theme_enabled() and not is_microsite():
-    header_extra_file = 'theme-head-extra.html'
```

Comment:
It looks like this change drops support for `theme-head-extra.html`. The README should mention that `theme-head-extra.html` should be renamed to `header-extra.html` (or restore support for `theme-head-extra.html`).